modules/sandboxed-containers-rhcos-extensions.adoc

```diff
@@ -1,10 +1,12 @@
 //Module included in the following assemblies:
 //
-// * sandboxed_containers/understanding_sandboxed_containers.adoc
+// * sandboxed_containers/understanding_sandboxed_containers-workloads.adoc
 
 :_content-type: CONCEPT
 [id="sandboxed-containers-rhcos-extensions_{context}"]
 
 = {op-system} extensions
 
-The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The sandboxed containers {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
+The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. {op-system-first} extensions are a mechanism to install optional {product-title} software. The {sandboxed-containers-operator} uses this mechanism to deploy sandboxed containers on a cluster.
+
+The sandboxed containers {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```

Comment:
@c3d. regarding the above sentence and comment:

- These acronyms are used without explanation throughout the document. I can either leave as is or change to something like: "The sandboxed containers RHCOS extension contains RPM packages for Kata, QEMU (Quick Emulator), and its dependencies." whichever is more correct, so please let me know.

- What does "its dependencies" relate to - the dependencies of the extension or of the QEMU?

- I am assuming “You can enable them” refers to enabling the RPMs – correct?

Reply:
@kelbrown20 Decided to leave as is because the acronyms appear like this across the documentation and the other questions relate to text that was there in previous versions.